### ISSUE TITLE
Weird scrollbar behavior

### ISSUE THREAD

**Issue by slorber** [henrybuilt/react-sticky-table#47]
I'm at the end of the table, and the scrollbar is not positionned correctly at the end (on Chome + IE11)

![image](https://user-images.githubusercontent.com/749374/44517735-c8f57680-a6c8-11e8-903c-ed15e06187a0.png)

Code is pretty simple:

```jsx
const StickerHeaderTable = ({ headers, rows }) => (
  <div css={{ width: '100%', height: '100%', border: 'solid thin yellow' }}>
    <StickyTable stickyColumnCount={0} stickyHeaderCount={1}>
      <TableRow>
        {headers.map((header, i) => (
          <Cell key={i}>{header}</Cell>
        ))}
      </TableRow>
      {rows.map((row, i) => (
        <TableRow key={i}>
          {row.map((cell, j) => (
            <Cell key={j}>{cell}</Cell>
          ))}
        </TableRow>
      ))}
    </StickyTable>
  </div>
);

 <StickerHeaderTable
        headers={[
          <div css={{ padding: 20, minWidth: 500 }}>column A</div>,
          <div css={{ padding: 20, minWidth: 500 }}>column B</div>,
          <div css={{ padding: 20, minWidth: 500 }}>column C</div>,
        ]}
        rows={[
          ['first', '1 b', '1 c'],
          ['2 a', '2 b', '2 c'],
          ['3 a', '3 b', '3 c'],
          ['4 a', '4 b', '4 c'],
          ['5 a', '5 b', '5 c'],
          ['6 a', '6 b', '6 c'],
          ['6 a', '6 b', '6 c'],
          ['6 a', '6 b', '6 c'],
          ['6 a', '6 b', '6 c'],
          ['6 a', '6 b', '6 c'],
          ['6 a', '6 b', '6 c'],
          ['6 a', '6 b', '6 c'],
          ['6 a', '6 b', '6 c'],
          ['6 a', '6 b', '6 c'],
          ['6 a', '6 b', '6 c'],
          ['6 a', '6 b', '6 c'],
          ['last', '6 b', '6 c'],
        ]}
      />
```

**Comment by slorber** [comment]
No went with a custom solution instead which is not perfect but good enough for my usecase and not requiring any JS:

https://stackoverflow.com/questions/52000363/flexbox-row-with-inline-flex-fixed-item-width-fixing-flex-basis-bug-not-wo

**Comment by dominikdosoudil** [comment]
#46 is same probably

**Comment by maxhudson** [comment]
@slorber @dominikdosoudil thanks for letting me know. They are the same issue. I think it's relatively straightforward to fix, I just haven't had the time. Happy to accept a correct PR. Hopefully I can get to it soon

**Comment by dominikdosoudil** [comment]
@slorber I've made a pull request, would be great if you test if it works also in your project before the PR is checked by @maxhudson. Could you do it or you are already using something absolutely different?
To install sticky-table from fork:

`npm install --save dominikdosoudil/react-sticky-table`

**Comment by slorber** [comment]
sorry I've moved to a custom CSS only solution that works for me, I'm not using this lib anymore

**Comment by dominikdosoudil** [comment]
What about the PR?

**Comment by maxhudson** [comment]
resolved in v3.0.1
